Expecto.Tests/Tests.fs

```diff
@@ -459,6 +459,85 @@ let expecto =
           assertTestFailsWithMsg msg (test, Normal)
       ]
 
+      testList "#distributed" [
+        testCase "identical sequence" <| fun _ ->
+          Expect.distributed [|21;37|] <| Map [
+              (21, 1)
+              (37, 1)
+            ] <| "Identical"
+
+        testCase "sequence contains all in different order" <| fun _ ->
+          Expect.distributed [|21;37|] <| Map[
+              (37, 1)
+              (21, 1)
+            ]
+                             <| "Same elements in different order"
+
+        testCase "sequence doesn't contain repeats in expected" <| fun _ ->
+          let format = "Sequence should contain one, two and four"
+          let test () =
+            Expect.distributed [|2; 2; 4|]  <| Map [
+                  (2, 1)
+                  (1, 1)
+                  (4, 2)
+                ] <| format
+          let msg =
+            sprintf "%s.
+    Sequence `actual` does not contain all `expected` elements.
+        All elements in `actual`:
+        {2, 2, 4}
+        All elements in `expected` ['item', 'number of expected occurances']:
+        {[1, 1], [2, 1], [4, 2]}
```

Comment:
How about replacing `{[1, 1], [2, 1], [4, 2]}` with output `{1: 1, 2: 1, 4: 2}` just to remove a bit of cruft?